runtime/js/40_http.js

```diff
@@ -9,15 +9,6 @@
   const core = window.Deno.core;
   const { ReadableStream } = window.__bootstrap.streams;
 
-  function flatEntries(obj) {
-    const entries = [];
-    for (const key in obj) {
-      entries.push(key);
-      entries.push(obj[key]);
-    }
-    return entries;
-  }
-
```

Comment:
@nayeemrmn this function was used intentionally - flamegraphs shown that it's way faster than `Array.flat()`. If that's possible please revert